### ISSUE TITLE
New Boutiques Integration and File arrays/list

### ISSUE THREAD

**Issue by MontrealSergiy** [aces/cbrain#1193]
The old cbrain boutiques integrator supports file lists input parameter, see example below 

``` 
        {
            "name":              "Optional multi input 2",
            "id":                "minput2",
            "description":       "Any file or directory in CBRAIN, or even a CbrainFileList",
            "type":              "File",
            "optional":          true,
            "list":              true,
            "value-key":         "[MINPUT2]"
        }
```

(the case is different from usual use of CBCSV list, since the file list expanded in command and tool can be executed sequentially without any parallelism).

The new boutiques integration one, however, leads to Failure on Setup. Stack trace 

```
Exception raised while setting up: NameError: undefined local variable or method 
`cbrainfilelist' for #<BoutiquesTask::NewMultiBoutiquesDemo:0x00000007089a38>
[2021-12-17 17:18:31 UTC] Bourreau/app/models/boutiques_cluster_task.rb:92:in `block in setup'
[2021-12-17 17:18:31 UTC] Bourreau/app/models/boutiques_cluster_task.rb:79:in `each'
[2021-12-17 17:18:31 UTC] Bourreau/app/models/boutiques_cluster_task.rb:79:in `setup'
[2021-12-17 17:18:31 UTC] Bourreau/app/models/cluster_task.rb:741:in `block (2 levels) in setup_and_submit_job'
[2021-12-17 17:18:31 UTC] Bourreau/app/models/cluster_task.rb:740:in `chdir'
[2021-12-17 17:18:31 UTC] Bourreau/app/models/cluster_task.rb:740:in `block in setup_and_submit_job'
[2021-12-17 17:18:31 UTC] Bourreau/app/models/cluster_task.rb:1785:in `block (3 levels) in apply_tool_config_environment'
[2021-12-17 17:18:31 UTC] Bourreau/app/models/tool_config.rb:129:in `apply_environment'
[2021-12-17 17:18:31 UTC] Bourreau/app/models/cluster_task.rb:1784:in `block (2 levels) in apply_tool_config_environment'
[2021-12-17 17:18:31 UTC] Bourreau/app/models/tool_config.rb:129:in `apply_environment'
[2021-12-17 17:18:31 UTC] Bourreau/app/models/cluster_task.rb:1783:in `block in apply_tool_config_environment'
[2021-12-17 17:18:31 UTC] Bourreau/app/models/tool_config.rb:129:in `apply_environment'
[2021-12-17 17:18:31 UTC] Bourreau/app/models/cluster_task.rb:1782:in `apply_tool_config_environment'
[2021-12-17 17:18:31 UTC] Bourreau/app/models/cluster_task.rb:739:in `setup_and_submit_job'
[2021-12-17 17:18:31 UTC] Bourreau/app/models/bourreau_worker.rb:330:in `process_task'
```

**Comment by prioux** [comment]
That's actually a bug.

**Comment by prioux** [comment]
Will be fixed as part of #1199
